interviewing/whiteboarding.md

```diff
@@ -110,3 +112,11 @@ Write an input example, and walk it through the function's process and discuss i
 	- [Code Wars](https://www.codewars.com/)
 - Other resources for practicing
 	- [JS Fiddle](https://jsfiddle.net/) - helpful for practicing frontend coding with JavaScript, HTML and CSS
+
+### Check for Understanding
+
+- What is Whiteboard coding?
+- Why is Google Jamboard used?
```

Comment:
This question will not check any understanding of Whiteboard.

Reply:
Ok @Kundan28, I removed that question.